test/test_endtoend.py

```diff
@@ -187,8 +202,11 @@ class EndToEndHyBiTest(EndToEndTestBase):
     def setUp(self):
         EndToEndTestBase.setUp(self)
 
-    def _run_test_with_client_options(self, test_function, options):
-        server = self._run_server()
+    def _run_test_with_client_options(self,
```

Comment:
Since this function now also supports server options, how about renaming it `_run_test_with_options()` ?